package.json

```diff
@@ -22,10 +22,10 @@
     "integration": "testcafe chrome src/**/*.spec.ts",
     "integration:single": "testcafe chrome -L",
     "integration:headless": "testcafe --config-file \"integration-headless.testcaferc.json\"",
-    "integration:title-transfers": "node --experimental-modules ./integration/local/index.mjs",
-    "integration:dappeteer": "run-s wait blockchain:contracts integration:title-transfers",
+    "integration:metamask": "run-s wait blockchain:contracts integration:synpress",
+    "integration:synpress": "cross-env SECRET_WORDS='indicate, swing, place, chair, flight, used, hammer, soon, photo, region, volume, shuffle' NETWORK_NAME='localhost' synpress run -cf synpress.json -c supportFile='tests/e2e/support/index.js',fixturesFolder='tests/e2e/fixtures'",
```

Comment:
i think some params can be set in `synpress.json`, shall we move them to there instead?

Reply:
also, is there any configuration to run concurrent tests in ci/cd?

Reply:
for some reason the params are not loading when set in `synpress.json`, so thats y i have them here.
I have tried the configuration to run concurrently but it doesn't seem to be working.

Reply:
https://docs.cypress.io/guides/guides/parallelization#Turning-on-parallelization <- we can do there separately

so far how long does the entire ci/cd flow takes currently with this pr?

Reply:
about 10-15 min i think